test/addons-napi/test_make_callback/binding.cc

```diff
@@ -0,0 +1,48 @@
+#include <node_api.h>
+#include "../common.h"
+#include <vector>
+
+namespace {
+
+napi_value MakeCallback(napi_env env, napi_callback_info info) {
+  constexpr int kMaxArgs = 10;
+  size_t argc = kMaxArgs;
+  napi_value args[kMaxArgs];
+  NAPI_CALL(env, napi_get_cb_info(env, info, &argc, args, NULL, NULL));
+
+  NAPI_ASSERT(env, argc > 0, "Wrong number of arguments");
+
+  napi_value recv = args[0];
+  napi_value func = args[1];
+
+  std::vector<napi_value> argv;
+  for (size_t n = 2; n < argc; n += 1) {
+    argv.push_back(args[n]);
+  }
+
+  napi_valuetype func_type;
+
+  NAPI_CALL(env, napi_typeof(env, args[1], &func_type));
```

Comment:
oversight - I'll change (initially everything was args[blah] but I refactored to use locals but missed updating this reference)